src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -140,7 +140,8 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
 
     // preconditions
     mdFile.contentAsString shouldBe "{}"
-    oldDepositProperties should not include "identifier.doi"
+    oldDepositProperties should include("identifier.doi.registered = no")
```

Comment:
Besides the fact that I would rather see this `depositPropertiesFile` be read as a `PropertiesConfiguration` rather than a `String` and doing `props.getString(...) shouldBe ...` (@jo-pol, I'm looking at you 😉), I am wondering what this check adds to this test. You're testing that it says `identifier.doi.registered=no`, but maybe there is a better place for this, as it seems a bit out of context in this test. Maybe @jo-pol can point you to a nice place to put this line?